libs/elements/src/lib/select/stories/vl-select.stories-arg.ts

```diff
@@ -1,11 +1,13 @@
 import { CATEGORIES, TYPES } from '@domg-wc/common-storybook';
 import { ArgTypes } from '@storybook/web-components';
+import { SELECT_POSITION } from '../vl-select.element';
 
 export const selectArgs = {
     block: false,
     disabled: false,
     error: false,
     noMoreOptions: 'Geen resterende opties gevonden',
```

Comment:
dit is niet het stukje code waar je nu op werkt maar vind het goed wat je deed voor `SELECT_POISITION`

mss goed dat we in toekomst ook kijken dat waardes die default zijn het component, direct van component kunnen komen, is beetje minder prone to human error en dan is er ook maar 1 plaats waar die aangepast moeten worden

meer specifiek zou dit hier kunnen voor noMoreOptions, searchNoResultsText, searchPlaceholder